package.json

```diff
@@ -2,7 +2,7 @@
   "name": "cozy-desktop",
   "version": "0.16.0",
   "description": "File Synchronization Client for Cozy Cloud",
-  "homepage": "https://github.com/cozy-labs/cozy-desktop",
+  "homepage": "https://docs.cozy.io/en/mobile/desktop.html",
```

Comment:
We try to keep documentation for users on cozy.io, and on github for people who want to contribute to our projects